homeassistant/components/todo/__init__.py

```diff
@@ -537,3 +543,12 @@ async def _async_remove_completed_items(entity: TodoListEntity, _: ServiceCall)
     ]
     if uids:
         await entity.async_delete_todo_items(uids=uids)
+
+
+async def _async_reset_completed_items(entity: TodoListEntity, _: ServiceCall) -> None:
```

Comment:
Is it worth having optional data that is a list of items so it's possible to reset a subset? (possibly as a future PR)